docs/index.rst

```diff
@@ -6,7 +6,7 @@ Vyper
     :alt: Vyper logo
     :align: center
 
-Vyper is a contract-oriented, pythonic programming language that targets the `Ethereum Virtual Machine (EVM) <http://ethdocs.org/en/latest/introduction/what-is-ethereum.html#ethereum-virtual-machine>`_
+Vyper is a contract-oriented, pythonic programming language that targets the `Ethereum Virtual Machine (EVM) <https://ethereum-homestead.readthedocs.io/en/latest/introduction/what-is-ethereum.html#ethereum-virtual-machine>`_.
```

Comment:
Can't we link to an article on ethereum.org or ethhub.io?

Reply:
one of these maybe?

[Solidity's version](https://solidity.readthedocs.io/en/v0.6.0/introduction-to-smart-contracts.html#index-6) - I think they paraphrased the homestead docs
[the EVM awesome list](https://github.com/ethereum/wiki/wiki/Ethereum-Virtual-Machine-(EVM)-Awesome-List) - a big list, let people find their own resources
[Ethereum in depth](https://blog.openzeppelin.com/ethereum-in-depth-part-1-968981e6f833/) - some good content by zeppelin

Reply:
Maybe something like https://ethereum.org/learn/#ethereum-basics or https://docs.ethhub.io/ethereum-basics/what-is-ethereum/